state/accountsDB.go

```diff
@@ -156,12 +156,16 @@ func checkArgsAccountsDB(args ArgsAccountsDB) error {
 	return nil
 }
 
-func startSnapshotAfterRestart(adb AccountsAdapter, tsm common.StorageManager) {
+func startSnapshotAfterRestart(adb AccountsAdapter, args ArgsAccountsDB) {
+	tsm := args.Trie.GetStorageManager()
 	epoch, err := tsm.GetLatestStorageEpoch()
 	if err != nil {
 		log.Error("could not get latest storage epoch")
 	}
-	if epoch == 0 && err == nil {
+	putActiveDBMarker := epoch == 0 && err == nil
+	putActiveDBMarker = putActiveDBMarker || args.ProcessingMode == common.ImportDb
```

Comment:
might add a new variable. for example `isInImportDBMode` and then check if either of them is true

Reply:
done